src/lib/wizard/components/shared/StatusIcon.svelte

```diff
@@ -0,0 +1,19 @@
+<script lang="ts">
+  type Props = {
+    type: "success" | "error" | "warn" | "info" | "loading";
+  };
+
+  let { type }: Props = $props();
+</script>
+
+{#if type === "success"}
+  <i class={`fa fa-check-circle-o text-green-600`}></i>
+{:else if type === "error"}
+  <i class={`fa fa-times-circle-o text-red-600`}></i>
+{:else if type === "warn"}
```

Comment:
We could use a different icon for the form statuses, since the warning triangle clashes with warning messages that are used in the forms themselves.

This currently appears as:
![Screenshot 2025-02-24 at 5 37 38 PM](https://github.com/user-attachments/assets/0bfa59c0-3f09-4a2c-9431-869860a38a01)

For example, add another condition like
```
{:else if type === "pending"}
  <i class={`fa fa-circle-o text-yellow-600`}></i>
```

So it becomes like:
![Screenshot 2025-02-24 at 5 44 32 PM](https://github.com/user-attachments/assets/4190d720-27a5-4224-a81b-abec65903356)